packages/back-end/src/controllers/features.ts

```diff
@@ -143,6 +146,20 @@ async function getPayloadParamsFromApiKey(
 }
 
 export async function getFeaturesPublic(req: Request, res: Response) {
+  // Cloud only: Invoke GB for SSE rollout rules
+  let growthbook: GrowthBook | null = null;
+  if (IS_CLOUD) {
+    growthbook = new GrowthBook<AppFeatures>({
+      apiHost: "https://cdn.growthbook.io",
+      clientKey:
+        process.env.NODE_ENV === "production"
+          ? "sdk-ueFMOgZ2daLa0M"
+          : "sdk-UmQ03OkUDAu7Aox",
+      enableDevMode: true,
+    });
+    await growthbook.loadFeatures({ autoRefresh: false });
```

Comment:
This works locally. Do we need to use a Fetch polyfill?

I also considered pulling from mongo, but not sure if it's worth the effort given this is usually CDN cached.

Reply:
any advice for avoiding the infinite CDN->features loop? Do you suggest manually building the payload from mongo when its our own?

Reply:
any advice for avoiding the infinite CDN->features loop? Do you suggest manually building the payload from mongo when its our own?